app/dashboard/embed.py

```diff
@@ -200,64 +202,102 @@ def embed(request):
         return err_response
 
 
-def avatar(request):
+def get_avatar(_org_name):
```

Comment:
Do you mind if I modify this to use an S3 storage backend versus local storage?

Reply:
fine with me.. maybe worth a v2 ticket?

Reply:
@owocki #912